website/content/docs/platform/k8s/vso/sources/vault.mdx

```diff
@@ -0,0 +1,283 @@
+| [AWS](/vault/docs/auth/aws)               | Relies on AWS credentials for Vault authentication                                                          |
+
+## Vault access and custom resource definitions
+
+The Vault connection and authentication configuration is provided by the `VaultConnection` and `VaultAuth` CRDs. These can be considered as
+foundational Custom Resources that all secret replication type resources will reference.
+
+### VaultConnection custom resource
+
+Provides the configuration necessary for the Operator to connect to a single Vault server instance.
+
+```yaml
+---
+apiVersion: secrets.hashicorp.com/v1beta1
+kind: VaultConnection
+metadata:
+  namespace: vso-example
+  name: vault-connection
+spec:
+  # required configuration
+  # address to the Vault server.
+  address: http://vault.vault.svc.cluster.local:8200
+
+  # optional configuration
+  # HTTP headers to be included in all Vault requests.
+  # headers: []
+  # TLS server name to use as the SNI host for TLS connections.
+  # tlsServerName: ""
+  # skip TLS verification for TLS connections to Vault.
+  # skipTLSVerify: false
+  # the trusted PEM encoded CA certificate chain stored in a Kubernetes Secret
+  # caCertSecretRef: ""
+```
+
+### VaultAuth custom resource
+
+Provide the configuration necessary for the Operator to authenticate to a single Vault server instance as
+specified in a `VaultConnection` Custom Resource.
+
+```yaml
+---
+apiVersion: secrets.hashicorp.com/v1beta1
+kind: VaultAuth
+metadata:
+  namespace: vso-example
+  name: vault-auth
+spec:
+  # required configuration
+  # VaultConnectionRef of the corresponding VaultConnection CustomResource.
+  # If no value is specified the Operator will default to the `default` VaultConnection,
+  # configured in its own Kubernetes namespace.
+  vaultConnectionRef: vault-connection
+  # Method to use when authenticating to Vault.
+  method: kubernetes
+  # Mount to use when authenticating to auth method.
+  mount: kubernetes
+  # Kubernetes specific auth configuration, requires that the Method be set to kubernetes.
+  kubernetes:
+    # role to use when authenticating to Vault
+    role: example
+    # ServiceAccount to use when authenticating to Vault
+    # it is recommended to always provide a unique serviceAccount per Pod/application
+    serviceAccount: default
+
+  # optional configuration
+  # Vault namespace where the auth backend is mounted (requires Vault Enterprise)
+  # namespace: ""
+  # Params to use when authenticating to Vault
+  # params: []
+  # HTTP headers to be included in all Vault authentication requests.
+  # headers: []
+```
+
+## Vault secret custom resource definitions
+
+Provide the configuration necessary for the Operator to replicate a single Vault Secret to a single Kubernetes Secret.
+Each supported CRD is specialized to a *class* of Vault secret, documented below.
```

Comment:
```suggestion
Configure the secrets operator to replicate a single Vault Secret to a single Kubernetes Secret. Each supported CRD is specialized to a Vault secrets
 **class**.
```

Style correction: write in active voice

Reply:
Rather than referencing content with "documented below", we should either provide the explicit heading or use an in-doc link to take the reader to the right place